plugin/evm/warp_backend.go

```diff
@@ -0,0 +1,82 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/cache"
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/database/prefixdb"
+	"github.com/ava-labs/avalanchego/database/versiondb"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/utils/hashing"
+	"github.com/ava-labs/avalanchego/vms/platformvm/teleporter"
+)
+
+var (
+	_ WarpBackend = &warpBackend{}
+
+	dbPrefix = []byte("warp")
```

Comment:
let's remove this prefix in favor of taking in a database that we expect to already be prefixed as needed

Reply:
Once this is removed, we should make this var block into a single line since there will only be one variable being declared here.